From: Michael Biebl <biebl@debian.org>
Date: Tue, 18 Mar 2014 23:04:54 +0100
Subject: Fix underlinking in examples

---
 examples/libtracker-sparql/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/examples/libtracker-sparql/Makefile.am b/examples/libtracker-sparql/Makefile.am
index e4bcc89..c29d470 100644
--- a/examples/libtracker-sparql/Makefile.am
+++ b/examples/libtracker-sparql/Makefile.am
@@ -5,6 +5,7 @@ AM_CPPFLAGS = $(BUILD_CFLAGS) \
 	$(LIBTRACKER_SPARQL_CFLAGS)
 
 LDADD = $(top_builddir)/src/libtracker-sparql-backend/libtracker-sparql-$(TRACKER_API_VERSION).la \
+	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(LIBTRACKER_SPARQL_LIBS)
 
