Index: tracker/src/libtracker-extract/Makefile.am
===================================================================
--- tracker.orig/src/libtracker-extract/Makefile.am	2014-02-22 03:21:51.809975300 +0100
+++ tracker/src/libtracker-extract/Makefile.am	2014-02-22 03:21:51.805975247 +0100
@@ -7,15 +7,16 @@
 	-DTRACKER_EXTRACTORS_DIR=\""$(extractmodulesdir)"\" \
 	$(LIBTRACKER_EXTRACT_CFLAGS)
 
-lib_LTLIBRARIES = libtracker-extract-@TRACKER_API_VERSION@.la
+libtracker_extractdir = $(libdir)/tracker-$(TRACKER_API_VERSION)
+libtracker_extract_LTLIBRARIES = libtracker-extract.la
 
-libtracker_extract_@TRACKER_API_VERSION@_la_LIBADD =   \
+libtracker_extract_la_LIBADD =   \
 	$(top_builddir)/src/libtracker-sparql-backend/libtracker-sparql-@TRACKER_API_VERSION@.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS)                                  \
 	$(LIBTRACKER_EXTRACT_LIBS)
 
-libtracker_extract_@TRACKER_API_VERSION@_la_SOURCES =  \
+libtracker_extract_la_SOURCES =  \
 	tracker-data.h                                 \
 	tracker-encoding.c                             \
 	tracker-encoding.h                             \
@@ -51,60 +52,27 @@
 	tracker-xmp.h
 
 if HAVE_ENCA
-libtracker_extract_@TRACKER_API_VERSION@_la_SOURCES += \
+libtracker_extract_la_SOURCES += \
 	tracker-encoding-enca.c                        \
 	tracker-encoding-enca.h
 endif
 
 if HAVE_MEEGOTOUCH
-libtracker_extract_@TRACKER_API_VERSION@_la_SOURCES += \
+libtracker_extract_la_SOURCES += \
 	tracker-encoding-meegotouch.cpp                \
 	tracker-encoding-meegotouch.h
 endif
 
 if HAVE_LIBICU
-libtracker_extract_@TRACKER_API_VERSION@_la_SOURCES += \
+libtracker_extract_la_SOURCES += \
 	tracker-encoding-libicu.c                      \
 	tracker-encoding-libicu.h
 AM_CPPFLAGS +=                                         \
 	$(LIBICU_CFLAGS)
-libtracker_extract_@TRACKER_API_VERSION@_la_LIBADD +=  \
+libtracker_extract_la_LIBADD +=  \
 	$(LIBICU_LIBS)
 endif
 
-libtracker_extract_@TRACKER_API_VERSION@_la_LDFLAGS =  \
-	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) \
-	-export-symbols-regex '^tracker_.*'
-
-# Introspection foo
--include $(INTROSPECTION_MAKEFILE)
-INTROSPECTION_GIRS =
-INTROSPECTION_SCANNER_ARGS = \
-	--warn-all \
-	-I $(top_srcdir)/src \
-	-D TRACKER_COMPILATION \
-	--c-include="libtracker-extract/tracker-extract.h"
-
-INTROSPECTION_COMPILER_ARGS = --includedir=$(top_srcdir)/src/libtracker-sparql
-
-if HAVE_INTROSPECTION
-INTROSPECTION_CFLAGS += $(AM_CPPFLAGS)
-introspection_sources = $(libtracker_extract_@TRACKER_API_VERSION@_la_SOURCES)
-TrackerExtract-@TRACKER_API_VERSION@.gir: $(INTROSPECTION_SCANNER) $(lib_LTLIBRARIES) Makefile
-TrackerExtract_@TRACKER_API_VERSION_UNDERSCORES@_gir_INCLUDES = GObject-2.0 GModule-2.0
-TrackerExtract_@TRACKER_API_VERSION_UNDERSCORES@_gir_SCANNERFLAGS = \
-	--include-uninstalled=$(top_srcdir)/src/libtracker-sparql/Tracker-$(TRACKER_API_VERSION).gir
-TrackerExtract_@TRACKER_API_VERSION_UNDERSCORES@_gir_CFLAGS = $(INCLUDES)
-TrackerExtract_@TRACKER_API_VERSION_UNDERSCORES@_gir_LIBS = $(lib_LTLIBRARIES)
-TrackerExtract_@TRACKER_API_VERSION_UNDERSCORES@_gir_FILES = $(introspection_sources)
-
-INTROSPECTION_GIRS += TrackerExtract-$(TRACKER_API_VERSION).gir
+libtracker_extract_la_LDFLAGS =  \
+	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
 
-girdir = $(datadir)/gir-1.0
-gir_DATA = $(INTROSPECTION_GIRS)
-
-typelibdir = $(libdir)/girepository-1.0
-typelib_DATA = $(INTROSPECTION_GIRS:.gir=.typelib)
-
-CLEANFILES = $(gir_DATA) $(typelib_DATA)
-endif
Index: tracker/src/miners/fs/Makefile.am
===================================================================
--- tracker.orig/src/miners/fs/Makefile.am	2014-02-21 01:38:54.348515597 +0100
+++ tracker/src/miners/fs/Makefile.am	2014-02-22 03:23:58.963636763 +0100
@@ -48,7 +48,7 @@
 tracker_miner_fs_LDADD =                               \
 	$(top_builddir)/src/libtracker-miner/libtracker-miner-@TRACKER_API_VERSION@.la \
 	$(top_builddir)/src/libtracker-sparql-backend/libtracker-sparql-@TRACKER_API_VERSION@.la \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-data/libtracker-data.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS)                                  \
Index: tracker/src/tracker-extract/Makefile.am
===================================================================
--- tracker.orig/src/tracker-extract/Makefile.am	2014-02-21 01:38:54.356515676 +0100
+++ tracker/src/tracker-extract/Makefile.am	2014-02-22 03:24:14.231836230 +0100
@@ -188,7 +188,7 @@
 libextract_abw_la_CFLAGS = $(TRACKER_EXTRACT_MODULES_CFLAGS)
 libextract_abw_la_LDFLAGS = $(module_flags)
 libextract_abw_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS)
@@ -198,7 +198,7 @@
 libextract_mp3_la_CFLAGS = $(TRACKER_EXTRACT_MODULES_CFLAGS)
 libextract_mp3_la_LDFLAGS = $(module_flags)
 libextract_mp3_la_LIBADD =  \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS)
@@ -208,7 +208,7 @@
 libextract_vorbis_la_CFLAGS = $(TRACKER_EXTRACT_MODULES_CFLAGS)
 libextract_vorbis_la_LDFLAGS = $(module_flags)
 libextract_vorbis_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
@@ -219,7 +219,7 @@
 libextract_flac_la_CFLAGS = $(TRACKER_EXTRACT_MODULES_CFLAGS)
 libextract_flac_la_LDFLAGS = $(module_flags)
 libextract_flac_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
@@ -230,7 +230,7 @@
 libextract_mplayer_la_CFLAGS = $(TRACKER_EXTRACT_MODULES_CFLAGS)
 libextract_mplayer_la_LDFLAGS = $(module_flags)
 libextract_mplayer_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS)
@@ -242,7 +242,7 @@
 	$(LIBGSF_CFLAGS)
 libextract_oasis_la_LDFLAGS = $(module_flags)
 libextract_oasis_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
@@ -255,7 +255,7 @@
 	$(LIBGSF_CFLAGS)
 libextract_epub_la_LDFLAGS = $(module_flags)
 libextract_epub_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
@@ -268,7 +268,7 @@
 	$(LIBPNG_CFLAGS)
 libextract_png_la_LDFLAGS = $(module_flags)
 libextract_png_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
@@ -280,7 +280,7 @@
 libextract_ps_la_CFLAGS = $(TRACKER_EXTRACT_MODULES_CFLAGS)
 libextract_ps_la_LDFLAGS = $(module_flags)
 libextract_ps_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS)
@@ -290,7 +290,7 @@
 libextract_totem_la_CFLAGS = $(TRACKER_EXTRACT_MODULES_CFLAGS)
 libextract_totem_la_LDFLAGS = $(module_flags)
 libextract_totem_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS)
@@ -300,7 +300,7 @@
 libextract_xmp_la_CFLAGS = $(TRACKER_EXTRACT_MODULES_CFLAGS)
 libextract_xmp_la_LDFLAGS = $(module_flags)
 libextract_xmp_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS)
 
@@ -311,7 +311,7 @@
 	$(LIBXML2_CFLAGS)
 libextract_html_la_LDFLAGS = $(module_flags)
 libextract_html_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
 	$(LIBXML2_LIBS)
@@ -323,7 +323,7 @@
 	$(LIBGSF_CFLAGS)
 libextract_msoffice_la_LDFLAGS = $(module_flags)
 libextract_msoffice_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
@@ -336,7 +336,7 @@
 	$(LIBGSF_CFLAGS)
 libextract_msoffice_xml_la_LDFLAGS = $(module_flags)
 libextract_msoffice_xml_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS)
@@ -348,7 +348,7 @@
 	$(LIBGXPS_CFLAGS)
 libextract_xps_la_LDFLAGS = $(module_flags)
 libextract_xps_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
@@ -361,7 +361,7 @@
 	$(POPPLER_CFLAGS)
 libextract_pdf_la_LDFLAGS = $(module_flags)
 libextract_pdf_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
@@ -373,7 +373,7 @@
 	$(TRACKER_EXTRACT_MODULES_CFLAGS)
 libextract_dvi_la_LDFLAGS = $(module_flags)
 libextract_dvi_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS)
@@ -389,7 +389,7 @@
 	$(LIBCUE_CFLAGS)
 libextract_gstreamer_la_LDFLAGS = $(module_flags)
 libextract_gstreamer_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
 	$(GSTREAMER_LIBS) \
@@ -414,7 +414,7 @@
 	$(XINE_CFLAGS)
 libextract_xine_la_LDFLAGS = $(module_flags)
 libextract_xine_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
@@ -427,7 +427,7 @@
 	$(LIBGIF_CFLAGS)
 libextract_gif_la_LDFLAGS = $(module_flags)
 libextract_gif_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
@@ -440,7 +440,7 @@
 	$(LIBJPEG_CFLAGS)
 libextract_jpeg_la_LDFLAGS = $(module_flags)
 libextract_jpeg_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
@@ -453,7 +453,7 @@
 	$(LIBTIFF_CFLAGS)
 libextract_tiff_la_LDFLAGS = $(module_flags)
 libextract_tiff_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
 	$(LIBTIFF_LIBS)
@@ -465,7 +465,7 @@
 	$(TOTEM_PL_PARSER_CFLAGS)
 libextract_playlist_la_LDFLAGS = $(module_flags)
 libextract_playlist_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
 	$(TOTEM_PL_PARSER_LIBS)
@@ -475,7 +475,7 @@
 libextract_text_la_CFLAGS = $(TRACKER_EXTRACT_MODULES_CFLAGS)
 libextract_text_la_LDFLAGS = $(module_flags)
 libextract_text_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS)
 
@@ -484,7 +484,7 @@
 libextract_icon_la_CFLAGS = $(TRACKER_EXTRACT_MODULES_CFLAGS)
 libextract_icon_la_LDFLAGS = $(module_flags)
 libextract_icon_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS)
 
@@ -496,7 +496,7 @@
 
 libextract_iso_la_LDFLAGS = $(module_flags)
 libextract_iso_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
 	$(LIBOSINFO_LIBS)
@@ -509,7 +509,7 @@
 	$(AVCODEC_CFLAGS)
 libextract_libav_la_LDFLAGS = $(module_flags)
 libextract_libav_la_LIBADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
 	$(TRACKER_EXTRACT_MODULES_LIBS) \
@@ -540,9 +540,9 @@
 	tracker-main.h
 
 tracker_extract_LDADD = \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
 	$(top_builddir)/src/libtracker-sparql-backend/libtracker-sparql-@TRACKER_API_VERSION@.la \
 	$(top_builddir)/src/libtracker-miner/libtracker-miner-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-data/libtracker-data.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(BUILD_LIBS) \
Index: tracker/tests/libtracker-extract/Makefile.am
===================================================================
--- tracker.orig/tests/libtracker-extract/Makefile.am	2014-02-21 01:38:54.404516152 +0100
+++ tracker/tests/libtracker-extract/Makefile.am	2014-02-22 03:23:40.083390098 +0100
@@ -37,7 +37,7 @@
 LDADD =                                                \
 	$(top_builddir)/tests/common/libtracker-testcommon.la \
 	$(top_builddir)/src/libtracker-common/libtracker-common.la \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-sparql-backend/libtracker-sparql-@TRACKER_API_VERSION@.la \
 	$(BUILD_LIBS)                                  \
 	$(LIBTRACKER_EXTRACT_LIBS)
Index: tracker/examples/libtracker-extract/Makefile.am
===================================================================
--- tracker.orig/examples/libtracker-extract/Makefile.am	2014-02-21 01:38:54.308515200 +0100
+++ tracker/examples/libtracker-extract/Makefile.am	2014-02-22 03:24:41.452191826 +0100
@@ -11,7 +11,7 @@
 libextract_mockup_la_SOURCES = tracker-extract-mockup.c
 libextract_mockup_la_LDFLAGS = $(module_flags)
 libextract_mockup_la_LIBADD =                          \
-	$(top_builddir)/src/libtracker-extract/libtracker-extract-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-extract/libtracker-extract.la \
 	$(top_builddir)/src/libtracker-sparql-backend/libtracker-sparql-@TRACKER_API_VERSION@.la \
 	$(BUILD_LIBS)                                  \
 	$(LIBTRACKER_EXTRACT_LIBS)
