Index: src/libtracker-gtk/Makefile.in
===================================================================
--- src/libtracker-gtk/Makefile.in	(Revision 985)
+++ src/libtracker-gtk/Makefile.in	(Arbeitskopie)
@@ -59,7 +59,8 @@
 libLTLIBRARIES_INSTALL = $(INSTALL)
 LTLIBRARIES = $(lib_LTLIBRARIES)
 am__DEPENDENCIES_1 =
-libtracker_gtk_la_DEPENDENCIES = $(am__DEPENDENCIES_1)
+libtracker_gtk_la_DEPENDENCIES = $(am__DEPENDENCIES_1) \
+	$(top_builddir)/src/libtracker/libtrackerclient.la
 am_libtracker_gtk_la_OBJECTS = tracker-utils.lo tracker-ui.lo \
 	tracker-metadata-tile.lo tracker-tag-bar.lo \
 	tracker-keyword-store.lo
@@ -351,7 +352,10 @@
 	tracker-tag-bar.c	\
 	tracker-keyword-store.c
 
-libtracker_gtk_la_LIBADD = $(LIBTRACKERGTK_LIBS)
+libtracker_gtk_la_LIBADD = \
+	$(top_builddir)/src/libtracker/libtrackerclient.la \
+	$(LIBTRACKERGTK_LIBS)
+
 libtracker_gtk_la_LDFLAGS = -version-info 0:0:0
 all: all-am
 
Index: src/libtracker-gtk/Makefile.am
===================================================================
--- src/libtracker-gtk/Makefile.am	(Revision 984)
+++ src/libtracker-gtk/Makefile.am	(Arbeitskopie)
@@ -23,6 +23,9 @@
 	tracker-tag-bar.c	\
 	tracker-keyword-store.c
 
-libtracker_gtk_la_LIBADD = $(LIBTRACKERGTK_LIBS)
+libtracker_gtk_la_LIBADD = \
+	$(top_builddir)/src/libtracker/libtrackerclient.la \
+	$(LIBTRACKERGTK_LIBS)
+
 libtracker_gtk_la_LDFLAGS = -version-info 0:0:0
 
