Index: tracker/filters/application/msword_filter
===================================================================
--- tracker.orig/filters/application/msword_filter	2009-02-10 09:53:37.000000000 +0100
+++ tracker/filters/application/msword_filter	2009-02-10 09:54:02.000000000 +0100
@@ -1,23 +1,22 @@
 #!/bin/sh
 
-tmpdir=${TMPDIR-/tmp}
-tmpdir="$tmpdir"/tmpdir.$$
+tmpdir=`mktemp -td tracker.msword_filter.XXXXXXXXXX` || exit 1
 
-tmptxtfile=`mktemp`
+tmptxtfile=`mktemp` || exit 1

-(umask 077 && mkdir "$tmpdir") || exit 1
-
-tmpfile="$tmpdir"/tmpfile.$$
+tmpfile="$tmpdir"/tmpfile
 
 cp "$1" "$tmpfile"
 
-# Change the working directory to $tmpdir which we can safely deleter later.
+# Change the working directory to $tmpdir which we can safely delete later.
 # Some distributions ship a version of wvText which extracts the image files
 # from the word document and places them in the current working directory.
 
 cd "$tmpdir"

-nice -n19 wvText "$tmpfile" "$tmptxtfile"
+nice -n19 wvText "$tmpfile" "$tmptxtfile" > /dev/null 2>&1
+
+cat "$tmptxtfile"

 rm -rf "$tmpdir"
 rm "$tmptxtfile"
