Index: src/trackerd/tracker-utils.c
===================================================================
--- src/trackerd/tracker-utils.c	(Revision 882)
+++ src/trackerd/tracker-utils.c	(Revision 889)
@@ -87,18 +87,18 @@
 
 
 static Matches tmap[] = {
-		{"da", "Danish"},
-		{"nl", "Dutch"},
-		{"en", "English"},
- 		{"fi", "Finnish"},
-		{"fr", "French"},
-		{"de", "German"},
-		{"it", "Italian"},
-		{"nb", "Norwegian"},
-		{"pt", "Portuguese"},
-		{"ru", "Russian"},
-		{"es", "Spanish"},
-		{"sv", "Swedish"},
+		{"da", "danish"},
+		{"nl", "dutch"},
+		{"en", "english"},
+ 		{"fi", "finnish"},
+		{"fr", "french"},
+		{"de", "german"},
+		{"it", "italian"},
+		{"nb", "norwegian"},
+		{"pt", "portuguese"},
+		{"ru", "russian"},
+		{"es", "spanish"},
+		{"sv", "swedish"},
 		{NULL, NULL},
 };
 
