diff --git a/src/plugins/evolution/tracker-evolution-plugin.c b/src/plugins/evolution/tracker-evolution-plugin.c
index 8e56d8d..b84e5ac 100644
--- a/src/plugins/evolution/tracker-evolution-plugin.c
+++ b/src/plugins/evolution/tracker-evolution-plugin.c
@@ -41,7 +41,6 @@
 #include <sqlite3.h>
 
 #include <camel/camel.h>
-#include <camel/camel-db.h>
 
 #include <mail/mail-config.h>
 #include <mail/mail-session.h>
@@ -53,10 +52,6 @@
 #include <libedataserver/e-account.h>
 #include <libedataserver/e-account-list.h>
 
-#ifdef HAVE_EDS_2_29_1
-#include <e-util/e-account-utils.h>
-#endif
-
 #include <libtracker-client/tracker.h>
 #include <libtracker-client/tracker-sparql-builder.h>
 
@@ -285,16 +280,8 @@ get_email_and_fullname (const gchar *line, gchar **email, gchar **fullname)
 static void
 folder_registry_free (FolderRegistry *registry)
 {
-#ifdef HAVE_EDS_2_31_2
 	g_signal_handler_disconnect (registry->folder, registry->hook_info->hook_id);
-#else
-	camel_object_remove_event (registry->folder, registry->hook_info->hook_id);
-#endif
-#ifdef HAVE_EDS_2_31_2
 	g_object_unref (registry->folder);
-#else
-	camel_object_unref (registry->folder);
-#endif
 	g_free (registry->hook_info->account_uri);
 	g_slice_free (OnSummaryChangedInfo, registry->hook_info);
 	g_slice_free (FolderRegistry, registry);
@@ -311,11 +298,7 @@ folder_registry_new (const gchar *account_uri,
 	registry->hook_info->account_uri = g_strdup (account_uri);
 	registry->hook_info->self = self; /* weak */
 	registry->hook_info->hook_id = 0;
-#ifdef HAVE_EDS_2_31_2
 	g_object_ref (folder);
-#else
-	camel_object_ref (folder);
-#endif
 	registry->folder = folder;
 
 	return registry;
@@ -1334,15 +1317,9 @@ register_on_get_folder (gchar *uri, CamelFolder *folder, gpointer user_data)
 		goto not_ready;
 	}
 
-#ifdef HAVE_EDS_2_31_2
 	hook_id = g_signal_connect (folder, "changed",
 				    G_CALLBACK (on_folder_summary_changed),
 				    registry->hook_info);
-#else
-	hook_id = camel_object_hook_event (folder, "folder_changed",
-	                                   CAMEL_CALLBACK (on_folder_summary_changed),
-	                                   registry->hook_info);
-#endif
 	registry->hook_info->hook_id = hook_id;
 
 	g_hash_table_replace (priv->registered_folders,
@@ -1519,11 +1496,7 @@ free_worker_thread_info (gpointer data, gpointer user_data)
 	/* Ownership was transfered to us in try_again */
 	free_introduction_info (winfo->intro_info);
 	camel_db_close (winfo->cdb_r);
-#ifdef HAVE_EDS_2_31_2
 	g_object_unref (winfo->store);
-#else
-	camel_object_unref (winfo->store);
-#endif
 	camel_folder_info_free (winfo->iter);
 	g_free (winfo);
 }
@@ -1578,11 +1551,7 @@ on_got_folderinfo_introduce (CamelStore *store,
 
 	/* Ownership of these is transfered in try_again */
 
-#ifdef HAVE_EDS_2_31_2
 	g_object_ref (store);
-#else
-	camel_object_ref (store);
-#endif
 	info->store = store;
 	/* This apparently creates a thread */
 	info->cdb_r = camel_db_clone (store->cdb_r, NULL);
@@ -1659,11 +1628,7 @@ introduce_account_to (TrackerEvolutionPlugin *self,
 
 	mail_get_folderinfo (store, NULL, on_got_folderinfo_introduce, intro_info);
 
-#ifdef HAVE_EDS_2_31_2
 	g_object_unref (store);
-#else
-	camel_object_unref (store);
-#endif
 
 }
 
@@ -1824,11 +1789,7 @@ on_folder_deleted (CamelStore *store,
 
 static void
 on_folder_renamed (CamelStore *store,
-#ifdef HAVE_EDS_2_31_2
 		   gchar *old_name,
-#else
-                   CamelRenameInfo *info,
-#endif
                    StoreRegistry *registry)
 {
 	unregister_account (registry->self, registry->account);
@@ -1846,11 +1807,7 @@ store_registry_new (gpointer co,
 	registry->store = co;
 	registry->account = account; /* weak */
 	registry->self = self; /* weak */
-#ifdef HAVE_EDS_2_31_2
 	g_object_ref (co);
-#else
-	camel_object_ref (co);
-#endif
 
 	return registry;
 }
@@ -1858,16 +1815,8 @@ store_registry_new (gpointer co,
 static void
 store_registry_free (StoreRegistry *registry)
 {
-#ifdef HAVE_EDS_2_31_2
 	g_signal_handler_disconnect (registry->store, registry->hook_id);
-#else
-	camel_object_remove_event (registry->store, registry->hook_id);
-#endif
-#ifdef HAVE_EDS_2_31_2
 	g_object_unref (registry->store);
-#else
-	camel_object_unref (registry->store);
-#endif
 	g_slice_free (StoreRegistry, registry);
 }
 
@@ -1895,45 +1844,27 @@ on_got_folderinfo_register (CamelStore *store,
 
 	/* Hook up catching folder changes in the store */
 	registry = store_registry_new (store, account, self);
-#ifdef HAVE_EDS_2_31_2
 	hook_id = g_signal_connect (store, "folder-created",
 				    G_CALLBACK (on_folder_created),
 				    registry);
-#else
-	hook_id = camel_object_hook_event (store, "folder_created",
-	                                   CAMEL_CALLBACK (on_folder_created),
-	                                   registry);
-#endif
 	registry->hook_id = hook_id;
 	g_hash_table_replace (priv->registered_stores,
 	                      GINT_TO_POINTER (hook_id),
 	                      registry);
 
 	registry = store_registry_new (store, account, self);
-#ifdef HAVE_EDS_2_31_2
 	hook_id = g_signal_connect (store, "folder-renamed",
 				    G_CALLBACK (on_folder_renamed),
 				    registry);
-#else
-	hook_id = camel_object_hook_event (store, "folder_renamed",
-	                                   CAMEL_CALLBACK (on_folder_renamed),
-	                                   registry);
-#endif
 	registry->hook_id = hook_id;
 	g_hash_table_replace (priv->registered_stores,
 	                      GINT_TO_POINTER (hook_id),
 	                      registry);
 
 	registry = store_registry_new (store, account, self);
-#ifdef HAVE_EDS_2_31_2
 	hook_id = g_signal_connect (store, "folder-deleted",
 				    G_CALLBACK (on_folder_deleted),
 				    registry);
-#else
-	hook_id = camel_object_hook_event (store, "folder_deleted",
-	                                   CAMEL_CALLBACK (on_folder_deleted),
-	                                   registry);
-#endif
 	registry->hook_id = hook_id;
 	g_hash_table_replace (priv->registered_stores,
 	                      GINT_TO_POINTER (hook_id),
@@ -1986,11 +1917,7 @@ register_account (TrackerEvolutionPlugin *self,
 	/* Get the account's folder-info and register it asynchronously */
 	mail_get_folderinfo (store, NULL, on_got_folderinfo_register, reg_info);
 
-#ifdef HAVE_EDS_2_31_2
 	g_object_unref (store);
-#else
-	camel_object_unref (store);
-#endif
 }
 
 static gboolean
@@ -2053,11 +1980,7 @@ unregister_account (TrackerEvolutionPlugin *self,
 	/* Get the account's folder-info and unregister asynchronously */
 	mail_get_folderinfo (store, NULL, on_got_folderinfo_unregister, reg_info);
 
-#ifdef HAVE_EDS_2_31_2
 	g_object_unref (store);
-#else
-	camel_object_unref (store);
-#endif
 }
 
 static void
@@ -2347,11 +2270,7 @@ tracker_evolution_plugin_init (TrackerEvolutionPlugin *plugin)
 	                         G_TYPE_STRING, G_TYPE_STRING, G_TYPE_STRING,
 	                         G_TYPE_INVALID);
 
-#ifdef HAVE_EDS_2_29_1
 	priv->accounts = g_object_ref (e_get_account_list ());
-#else
-	priv->accounts = g_object_ref (mail_config_get_accounts ());
-#endif
 
 	for (it = e_list_get_iterator (E_LIST (priv->accounts)); e_iterator_is_valid (it); e_iterator_next (it)) {
 		register_account (plugin, (EAccount *) e_iterator_get (it));
