Index: tracker/python/deskbar-handler/tracker-handler.py
===================================================================
--- tracker.orig/python/deskbar-handler/tracker-handler.py	2009-02-20 02:49:58.000000000 +0100
+++ tracker/python/deskbar-handler/tracker-handler.py	2009-03-06 16:20:55.000000000 +0100
@@ -21,12 +21,13 @@
 import urllib
 import string
 import gnome
+import gnomedesktop
 import gobject
 
 import gettext
 gettext.install('tracker')
 
-import deskbar, deskbar.Utils, deskbar.gnomedesktop
+import deskbar, deskbar.Utils
 import deskbar.Handler
 import deskbar.Match
 
@@ -345,13 +346,13 @@
 # this code is stolen from the programs handler of deskbar
 def parse_desktop_file(desktop, only_if_visible=False):
 	try:
-		desktop = deskbar.gnomedesktop.item_new_from_file(desktop, deskbar.gnomedesktop.LOAD_ONLY_IF_EXISTS)
+		desktop = gnomedesktop.item_new_from_file(desktop, gnomedesktop.LOAD_ONLY_IF_EXISTS)
 	except Exception, e:
 		print 'Couldn\'t read desktop file:%s:%s' % (desktop, e)
 		return None
-	if desktop == None or desktop.get_entry_type() != deskbar.gnomedesktop.TYPE_APPLICATION:
+	if desktop == None or desktop.get_entry_type() != gnomedesktop.TYPE_APPLICATION:
 		return None
-	if only_if_visible and desktop.get_boolean(deskbar.gnomedesktop.KEY_NO_DISPLAY):
+	if only_if_visible and desktop.get_boolean(gnomedesktop.KEY_NO_DISPLAY):
 		return None
 	return desktop
 
Index: tracker/python/deskbar-handler/tracker-module.py
===================================================================
--- tracker.orig/python/deskbar-handler/tracker-module.py	2009-03-06 16:20:54.000000000 +0100
+++ tracker/python/deskbar-handler/tracker-module.py	2009-03-06 16:20:55.000000000 +0100
@@ -1,6 +1,7 @@
 # This deskbar module was ported from deskbar <= 2.18 handler by Marcus Fritzsch
 
 import gnome
+import gnomedesktop
 import gobject
 import re
 import sys
@@ -11,7 +12,6 @@
 import os.path
 import deskbar
 import deskbar.core.Utils
-import deskbar.core.gnomedesktop
 import deskbar.interfaces.Module
 import deskbar.interfaces.Match
 import deskbar.interfaces.Action
@@ -470,13 +470,13 @@
 # this code is stolen from the programs handler of deskbar
 def parse_desktop_file(desktop, only_if_visible=False):
 	try:
-		desktop = deskbar.core.gnomedesktop.item_new_from_file(desktop, deskbar.core.gnomedesktop.LOAD_ONLY_IF_EXISTS)
+		desktop = gnomedesktop.item_new_from_file(desktop, gnomedesktop.LOAD_ONLY_IF_EXISTS)
 	except Exception, e:
 		print 'Couldn\'t read desktop file:%s:%s' % (desktop, e)
 		return None
-	if desktop == None or desktop.get_entry_type() != deskbar.core.gnomedesktop.TYPE_APPLICATION:
+	if desktop == None or desktop.get_entry_type() != gnomedesktop.TYPE_APPLICATION:
 		return None
-	if only_if_visible and desktop.get_boolean(deskbar.core.gnomedesktop.KEY_NO_DISPLAY):
+	if only_if_visible and desktop.get_boolean(gnomedesktop.KEY_NO_DISPLAY):
 		return None
 	return desktop
 
