Index: ChangeLog
===================================================================
--- ChangeLog	(revision 1078)
+++ ChangeLog	(working copy)
@@ -1,3 +1,13 @@
+2007-12-13 Jamie McCracken <jamiemcc at gnome org>
+
+	* Fixed issue with log file being null
+	* Fixed typo of metadata name Audio:PeakTrackGain in the gstreamer extractor
+	* fix for crasher where index name is null during merging
+	* make initial indexing on battery off by default
+	* Fixed prevention of multiple instances of trackerd causing infinite loop and 100% cpu
+	* Fixed reentrant signals to prevent loops by dying quickly
+
+
 2007-12-09 Jamie McCracken <jamiemcc at gnome org>
 
 	* Increased startup sleep for applet to prevent race condition
Index: src/trackerd/trackerd.c
===================================================================
--- src/trackerd/trackerd.c	(revision 1078)
+++ src/trackerd/trackerd.c	(working copy)
@@ -618,19 +618,13 @@
 static void
 signal_handler (gint signo)
 {
-	if (!tracker->is_running) {
-		return;
-	}
-
 	static gboolean in_loop = FALSE;
 
-  	/* avoid re-entrant signals handler calls */
-	if (in_loop && signo != SIGSEGV) {
-		return;
+  	/* die if we get re-entrant signals handler calls */
+	if (in_loop) {
+		exit (EXIT_FAILURE);
 	}
 
-  	in_loop = TRUE;
-
   	switch (signo) {
 
   		case SIGSEGV:
@@ -646,6 +640,8 @@
 		case SIGTERM:
 		case SIGINT:
 
+		  	in_loop = TRUE;
+
 			tracker->is_running = FALSE;
 			tracker_end_watching ();
 
@@ -660,8 +656,7 @@
 			if (tracker->log_file && g_strsignal (signo)) {
 	   			tracker_log ("Received signal %s ", g_strsignal (signo));
 			}
-			in_loop = FALSE;
-    			break;
+			break;
   	}
 }
 
@@ -2422,6 +2417,14 @@
 	if (!g_thread_supported ())
 		g_thread_init (NULL);
 
+	dbus_g_thread_init ();
+
+	tracker = g_new0 (Tracker, 1);
+
+	tracker->dbus_con = tracker_dbus_init ();
+	
+	add_local_dbus_connection_monitoring (tracker->dbus_con);
+
 	setlocale (LC_ALL, "");
 
 	bindtextdomain (GETTEXT_PACKAGE, TRACKER_LOCALEDIR);
@@ -2478,9 +2481,9 @@
 	sigaction (SIGINT,  &act, NULL);
 #endif
 
-	dbus_g_thread_init ();
+	
 
-	tracker = g_new0 (Tracker, 1);
+	
 
 	tracker->status = STATUS_INIT;
 
@@ -2506,9 +2509,7 @@
 
 	tracker_log ("starting log");
 
-	tracker->dbus_con = tracker_dbus_init ();
 	
-	add_local_dbus_connection_monitoring (tracker->dbus_con);
 
 
 	g_free (tmp_dir);
Index: src/trackerd/tracker-utils.c
===================================================================
--- src/trackerd/tracker-utils.c	(revision 1078)
+++ src/trackerd/tracker-utils.c	(working copy)
@@ -3491,11 +3491,13 @@
 	/* logging is thread safe */
 	static size_t   log_size = 0;
 
-	if (! message)
+	if (!message)
 		return;
 
 	g_print ("%s\n", message);
 
+	if (!tracker->log_file) return;
+
 	/* ensure file logging is thread safe */
 	g_mutex_lock (tracker->log_access_mutex);
 
