Index: src/tracker-search-tool/tracker-search-tool-support.h
===================================================================
--- src/tracker-search-tool/tracker-search-tool-support.h	(Revision 759)
+++ src/tracker-search-tool/tracker-search-tool-support.h	(Arbeitskopie)
@@ -121,6 +121,10 @@
 gboolean
 open_file_with_nautilus (GtkWidget * window,
                          const gchar * file);
+
+char *
+tracker_string_replace (const char *haystack, char *needle, char *replacement);
+
 gboolean
 open_file_with_application (GtkWidget * window,
                             const gchar * file);
@@ -139,7 +143,13 @@
 void
 tracker_search_set_columns_order (GtkTreeView * treeview);
 
+int
+tracker_get_stored_separator_position ();
+
 void
+tracker_set_stored_separator_position (int pos);
+
+void
 tracker_search_get_stored_window_geometry (gint * width,
                                         gint * height);
 
