Index: tracker/aclocal.m4
===================================================================
--- tracker.orig/aclocal.m4	2011-03-05 02:28:16.672365000 +0100
+++ tracker/aclocal.m4	2011-03-05 02:28:23.496951002 +0100
@@ -156,7 +156,7 @@
 Consider adjusting the PKG_CONFIG_PATH environment variable if you
 installed software in a non-standard prefix.
 
-_PKG_TEXT])dnl
+_PKG_TEXT])[]dnl
         ])
 elif test $pkg_failed = untried; then
      	AC_MSG_RESULT([no])
@@ -167,7 +167,7 @@
 
 _PKG_TEXT
 
-To get pkg-config, see <http://pkg-config.freedesktop.org/>.])dnl
+To get pkg-config, see <http://pkg-config.freedesktop.org/>.])[]dnl
         ])
 else
 	$1[]_CFLAGS=$pkg_cv_[]$1[]_CFLAGS
Index: tracker/configure
===================================================================
--- tracker.orig/configure	2011-03-05 02:28:16.708347000 +0100
+++ tracker/configure	2011-03-05 02:28:23.512943002 +0100
@@ -16432,10 +16432,9 @@
 LIBEXIF_REQUIRED=0.6
 LIBGSF_REQUIRED=1.13
 EXEMPI_REQUIRED=2.1.0
-EVO_REQUIRED=2.91.90
-EVO_SHELL_REQUIRED=2.91.90
-EDS_REQUIRED=2.91.90
-CAMEL_REQUIRED=2.91.90
+EVO_REQUIRED=2.31.2
+EDS_REQUIRED=2.31.2
+CAMEL_REQUIRED=2.31.2
 # Unlikely version for now, Nepomuk integration isn't finished in streamanalyzer atm
 LIBSTREAMANALYZER_REQUIRED=0.7.0
 GEE_REQUIRED=0.3
@@ -18353,7 +18352,6 @@
 evolution_plugin_name="evolution-plugin"
 
 TRACKER_MINER_EVOLUTION_REQUIRED="glib-2.0                  >= $GLIB_REQUIRED
-                                  evolution-shell           >= $EVO_SHELL_REQUIRED
                                   evolution-plugin          >= $EVO_REQUIRED
                                   evolution-data-server-1.2 >= $EDS_REQUIRED
                                   camel-1.2                 >= $CAMEL_REQUIRED"
